R/compare_two_years.R

```diff
@@ -32,9 +32,20 @@ compare_two_years <- function(w.use, data.elements, year.x.y, area.column, areas
   x <- gather_(w.use.sub, "Key", "Value", data.elements)
   
   for(i in data.elements){
+    
+    df_x<-x[x$YEAR == year.x.y[1] & x$Key == i,][["Value"]]
+    df_y<-x[x$YEAR == year.x.y[2] & x$Key == i,][["Value"]]
```

Comment:
Made these variables. We will need to line up area ids (fips codes) and cull out the counties that don't have matches if we want to support the case where there are different counties from one compilation year to another.